src/Microsoft.SqlTools.Serialization/Utility/CommandOptions.cs

```diff
@@ -0,0 +1,89 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace Microsoft.SqlTools.Serialization.Utility
+{
+    /// <summary>
+    /// The command-line options helper class.
+    /// </summary>
+    internal class CommandOptions
```

Comment:
Please refactor the original CommandOptions so we don't have 3 almost duplicate classes (this is a classic "Rule of 3" case where the 3rd time around you make sure to stop duplicating. Seems like the only difference is the name of the app in Usage, and that's not even correct for any of our exes since the .Net Core 2.0 move.

Reply:
Ended up refactoring CommandOptions by making it accept an additional argument to pass in the exe name. Moved CommandOptions to Hosting utilities since keeping it in e.g. service layer would create a circular dependency between credentials and service layer. Created a subclass of CommandOptions in service layer that overrode SetLocale. This was necessary to set the internal SR culture for service layer for service layer tests. I didn't want to do this in the base class since we would need to introduce an additional service layer dependency in hosting, which would definitely not be a good idea.